lib/azure/resource_management/ARM_interface.rb

```diff
@@ -250,10 +250,11 @@ def create_server(params = {})
         platform(params[:azure_image_reference_offer])
         ## resource group creation
         if resource_group_exist?(params[:azure_resource_group_name])
-          Chef::Log.info("Resource Group #{params[:azure_resource_group_name]} already exist. Skipping its creation.")
-          Chef::Log.info("Adding new VM #{params[:azure_vm_name]} to this resource group.")
+          ui.log("Resource Group #{params[:azure_resource_group_name]} already exist. Skipping its creation.")
+          ui.log("Adding new VM #{params[:azure_vm_name]} to this resource group.")
         else
-          Chef::Log.info("Creating ResourceGroup....")
+          ui.log("Creating ResourceGroup.... #{("\n")} ")

```

Comment:
Can we remove the additional #{("\n")} string from the log message string and directly write the character "\n" in the log message string if it serves the purpose.